.github/workflows/check_need_autoupgrade_pr_label.yml

```diff
@@ -0,0 +1,108 @@
+# This workflow allows you to check potential changes to the structure of the database,
+# and to apply the “Needs autoupgrade PR” label if this is the case
+name: Check the label 'Needs autoupgrade PR'
+
+on: pull_request_target
+
+jobs:
+  install_prestashop_and_dump_base:
+    name: Install Prestashop and create database dump for base branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
This works for now but I guess we'll have to update this so that it uses the PR target branch right?

Reply:
Since we are on the "pull_request_targe" event, we switch here to the repository of the destination branch. The change of repository which contains the PR changes is carried out on the other job, with 

```
- uses: actions/checkout@v4
  with:
    ref: ${{ github.event.pull_request.head.ref }}. 
```
          
For me there will be nothing to change